src/Libraries/RevitNodes/Properties/Resources.resx

```diff
@@ -117,10 +117,13 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="App_Description" xml:space="preserve">
```

Comment:
Yah, the use of `App_Description` has been moved from `Libraries/RevitNodes` to `DynamoRevit`, so a new entry is being added there.